### DISCUSSION TITLE
Logger component in LWC - Adding logger calls in renderedCallback LWC method results in other LWC's not rendering

### DISCUSSION THREAD

**Discussion by KumaranM2040** [jongpie/NebulaLogger#464] [Q&A] [answered]
I have a screen flow that displays a screen with multiple custom LWC components. I have added the Nebula LWC component to one of these custom LWCs. The logging in general works fine however I noticed that adding logger debug inside the renderedCallback resulted in other LWC components (That needed to be displayed after the LWC with logging) not being rendered in the screen. 

I then integrated the LoggerLWCDemo LWC into my screen with a few modifications (removed async apex logging call) with the main one being add the renderedCallback with logging:

  renderedCallback() {
        const logger = this.template.querySelector(LOGGER_NAME);
        logger.info(this.message);
        logger.saveLog();
    }

And this also showed the problem where all my subsequent Custom LWC were not rendered. Is this a known issue ? Do i need to use async logging in the rendered callback ?

**Comment by jongpie** [comment, marked as answer]
This is a known issue - #280. The PR #401 will address this, once it's been finalized & merged.
